tools/tools.go

```diff
@@ -368,17 +331,18 @@ func GetSecretValues(secrets []envmanModels.EnvironmentItemModel) []string {
 }
 
 // EnvmanRun runs a command through envman.
-func EnvmanRun(envstorePth,
+func EnvmanRun(envStorePth,
 	workDirPth string,
 	cmdArgs []string,
 	timeout time.Duration,
 	noOutputTimeout time.Duration,
 	secrets []string,
 	stdInPayload []byte,
 ) (int, error) {
-	logLevel := log.GetLevel().String()
-	args := []string{"--loglevel", logLevel, "--path", envstorePth, "run"}
-	args = append(args, cmdArgs...)
+	envs, err := envman.ReadAndEvaluateEnvs(envStorePth, &envmanEnv.DefaultEnvironmentSource{})
+	if err != nil {
+		return 1, err
+	}
```

Comment:
`envman.ReadAndEvaluateEnvs` results in `os.Setenv` with all the envs in the EnvStore + `os.Environ()`.

Because of this:
- the command envs always has the os.Environ
- no need to run the command through envman (`envman cmd args...` -> `cmd args...`)

Reply:
Cool, makes sense. Thanks for the explanation.